ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.11__okd.yaml

```diff
@@ -0,0 +1,72 @@
+base_images:
```

Comment:
Consider adding `.github` to the commit message. As per openshift-windows-machine-config-operator-community-4.11__okd.yaml [line #60](https://github.com/openshift/release/pull/31749/commits/b253e70e091d9e15d056c02d2df2f53037878d56#diff-8b833ec2e6e44eb563112a923b8aa8a4a9f7d10d1de096c94840be9697226c32R60)